legend-engine-xt-relationalStore-bigquery-pure/src/main/resources/core_relational_bigquery/relational/sqlQueryToString/bigQueryExtension.pure

```diff
@@ -36,11 +36,14 @@ function <<access.private>> meta::relational::functions::sqlQueryToString::bigQu
 
 function <<access.private>> meta::relational::functions::sqlQueryToString::bigQuery::getLiteralProcessorsForBigQuery():Map<Type,LiteralProcessor>[1]
 {
+   let literalReplacementPairForString = ^LiteralReplacement(old = '\'', new = '\\\'');
+
    newMap([
+      pair(String,         ^LiteralProcessor(format = '\'%s\'', transform = convertStringToSQLString($literalReplacementPairForString)->literalTransform(), literalReplacements = $literalReplacementPairForString)),
```

Comment:
can we use the same approach as here for spanner, postgres and extensionDefaults too?